src/server/v2/resources/images.py

```diff
@@ -83,6 +83,7 @@ def get(self):
         """ retrieve a list/collection of images """
         log_id = get_log_id()
         current_app.logger.info("%s ++ images.v2.GET", log_id)
+        current_app.logger.info("%s ++ images.v2.GET RAW:%s" % (log_id, current_app.data["images"].values()))
```

Comment:
Do we want to dump this much information into the logs? Maybe do this 'debug' level logging?